spec/integration/workflow-steps.feature

```diff
@@ -0,0 +1,46 @@
+ Feature: Workflow steps
+
+  The selected Step's properties can be viewed & some of them can be edited.
+  The selected Step's Inputs' properties can be viewed & some of them can be edited.
+  Steps of any Workflow are listed, they can be selected, and their name, version, icon & badges are displayed.
+
+  Background:
+    Given editor is open
+
+  Scenario: Step's selected
```

Comment:
I'm not a huge fan of cucumber, but thats just an opinion. Having used it in the past, I've found that when developers are writing the tests, they are more comfortable with using a programatic API, which can be a bit more flexible especially when trying to debug a test.

Reply:
I know it's controversial, we already have large number of test cases written in it (which are not automated) so it made sense for us to use it. I've used it as well many times, one advantage of this is readability and expressiveness over programatic apis. But we have underlying functions as well so if we think it's giving us pain we can get rid of this and use plain reusable functions quite easily.